snap/snapcraft.yaml

```diff
@@ -0,0 +1,25 @@
+name: bhttp
```

Comment:
You can specify `go-channel: latest/stable` (or something else) in the part that builds it.

It will use the go snap from the specified channel (by default `latest/stable`, which is currently 1.11.5)

You can see the available channels with `snap info go`

Reply:
As for the browser interaction, it's definitely possible, I still need to investigate why it's not working in this case